### PR TITLE
BGP API: fix search by ipv6 when using compressed addresses

### PR DESCRIPTION
**Pull request by kedare**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/html/api_functions.inc.php [hunk 1]
```diff
@@ -517,11 +518,11 @@ function list_bgp(\Illuminate\Http\Request $request)
     }
     if (!empty($local_address)) {
         $sql .= ' AND `bgpPeers`.`bgpLocalAddr` = ?';
-        $sql_params[] = $local_address;
+        $sql_params[] = IP::parse($local_address)->uncompressed();
```
Comment (murrant) [inline includes/html/api_functions.inc.php:521]: This will throw an exception if it isn't a valid IPv6 address

Comment (murrant) [reply includes/html/api_functions.inc.php:521]: best to put a try catch block around it and send an error to the client.

Comment (kedare) [reply includes/html/api_functions.inc.php]: Fixed :)

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-63-release-changelog-april-2020/11828/1

**Pull request merged**